bin/kick_off.sh

```
@@ -164,19 +164,27 @@ tests=0
 # Also chmod user.pem_test to be 500.
 #
 if [[ $ssh_key_file != "" ]]; then
-	cp $ssh_key_file config/user.pem_test
+	if [ ! -f $ssh_key_file ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)